README.md

```diff
@@ -1,30 +1,57 @@
-# GPU Sharing with Time Slicing, MPS, MIG and others
+# GPU Partitioning with Time Slicing, MPS, MIG and others
 
-Repository to demo GPU Sharing with Time Slicing, MPS, MIG and others with [Red Hat OpenShift AI](https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-ai) and [NVIDIA GPU Operator](https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/index.html).
+Repository to demo GPU Partitioning with Time Slicing, MPS, MIG and others with [Red Hat OpenShift AI](https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-ai) and [NVIDIA GPU Operator](https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/index.html).
 
-Check also the [OpenShift GPU Sharing Methods Docs](https://developer.nvidia.com/blog/improving-gpu-utilization-in-kubernetes/) if you want to know more.
+Check also the [OpenShift GPU Partitioning Methods Docs](https://developer.nvidia.com/blog/improving-gpu-utilization-in-kubernetes/) if you want to know more.
 
 ## Table of Contents
-- [GPU Sharing Overview](#1-gpu-sharing-overview)
+- [GPU Partitioning Overview](#1-gpu-partitioning-overview)
   - [Time-Slicing](#11-time-slicing)
   - [Multi-Instance GPU (MIG)](#12-multi-instance-gpu-mig)
   - [Multi-Process Service (MPS)](#13-multi-process-service-mps)
-- [Pros and Cons of GPU Sharing Methods](#14-pros-and-cons-of-gpu-sharing-methods)
+- [Pros and Cons of GPU Partitioning Methods](#14-pros-and-cons-of-gpu-partitioning-methods)
 - [Requirements](#2-requirements)
 - [Usage](#3-usage)
   - [Time-Slicing](#31-time-slicing)
   - [MIG-Single](#321-mig-single)
   - [MIG-Mixed](#322-mig-mixed)
   - [MPS](#33-mps)
-  - [NO GPU Sharing - Default](#34-no-gpu-sharing---default)
-- [Validate and Check GPU Sharing](#4-validate-and-check-gpu-sharing)
+  - [NO GPU Partitioning - Default](#34-no-gpu-partitioning---default)
+- [Validate and Check GPU Partitioning](#4-validate-and-check-gpu-partitioning)
 - [Testing with LLMs](#5-testing-with-llms)
 - [Install Nvidia GPU Operator from Staging / Development](#6-install-nvidia-gpu-operator-from-staging--development)
 - [Other Interesting Links](#7-other-interesting-links)
 
-## 1. GPU Sharing Overview
+## 0. Why GPU Partitioning?
 
-![GPU Sharing Overview](assets/gpu-sharing-overview.png)
+GPU partitioning is essential for optimizing the utilization of GPU resources in our OpenShift AI clusters. 
```

Comment:
I mostly agree, but to the un-educated eye, this sounds like "you must always do GPU partitioning" and/or "if you use OpenShift AI, you must do GPU partitioning".  We need to be 
1. less specific. (It's not just for OpenShift AI). 
2. less pushy. (if you have 1 GPU with 40GB of mem, and need to serve one LLM that will require 20-40 GB of mem, then you don't need to do GPU partitioning.)